FinSearchUnified/Controllers/Backend/FindologicStaging.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+use FinSearchUnified\Components\ConfigLoader;
+use Shopware\Components\CSRFWhitelistAware;
+use Shopware\Models\Config\Value;
+use Shopware\Models\Shop\Repository;
+use Shopware\Models\Shop\Shop;
+
+class Shopware_Controllers_Backend_FindologicStaging extends Shopware_Controllers_Backend_ExtJs implements
+    CSRFWhitelistAware
+{
+    public function indexAction()
+    {
+        try {
+            $cacheManager = Shopware()->Container()->get('shopware.cache_manager');
+            $cacheManager->clearConfigCache();
+
+            Shopware()->Plugins()->Controller()->ViewRenderer()->setNoRender(true);
+            $shopkey = Shopware()->Config()->offsetGet('ShopKey');
+            $isActive = Shopware()->Config()->offsetGet('ActivateFindologic');
+
+            if (!$isActive) {
+                throw new RuntimeException('Please ensure the plugin is active!');
+            }
+            if (!preg_match('/^[A-F0-9]{32}$/', $shopkey)) {
+                throw new RuntimeException('Please ensure a valid ShopKey is provided!');
```

Comment:
You have not imported the `RuntimeException` class.

Reply:
It does not need to be imported. It is from the default namespace.

Reply:
See my other comment.